app/src/components/ui/SearchBar.vue

```diff
@@ -1,16 +1,28 @@
 <template>
-    <v-combobox
-            style="background: white"
-            rounded
-            color="black"
-            placeholder="z.B.Jugendarbeit"
-            item-text="tag"
-            autocomplete="off"
-            :items="mySearchProposals"
-            @input="addSearchTag"
-            append-icon="none"
-            :search-input.sync="mySearchValue"
-    ></v-combobox>
+  <v-combobox
+    style="background: white"
+    rounded
+    color="black"
+    placeholder="z.B.Jugendarbeit"
+    item-text="tag"
+    autocomplete="off"
+    :items="mySearchProposals"
+    @input="addSearchTag"
+    append-icon="none"
+    :search-input.sync="mySearchValue"
+  >
+    <template v-slot:no-data v-if="mySearchValue.length > 0">
```

Comment:
`Error in render: "TypeError: _vm.mySearchValue is null"`
null has no attribute length. Please test for null values.